skxray/fitting/xrf_model.py

```diff
@@ -807,6 +816,98 @@ def setup_element_model(self, elemental_line, default_area=1e5):
                 else:
                     element_mod = gauss_mod
 
+        else:
+            logger.debug('Started setting up pileup peaks for {}'.format(
+                elemental_line))
+
+            element_line1, element_line2 = elemental_line.split('-')
+
+            def get_line_energy(elemental_line):
+                """Return the energy of the first line in K, L or M series.
+                Parameters
+                ----------
+                elemental_line : str
+                    For instance, Eu_L is the format for L lines and Pt_M for M lines.
+                    And for K lines, user needs to define lines like ka1, kb1,
+                    because for K lines, we consider contributions from either ka1
+                    or kb1, while for L or M lines, we only consider the primary peak.
+
+                Returns
+                -------
+                float :
+                    energy of emission line
+                """
+                name, line = elemental_line.split('_')
+                line = line.lower()
+                e = Element(name)
+                if 'k' in line:
+                    e_cen = e.emission_line[line]
+                elif 'l' in line:
+                    # only the first line for L
+                    e_cen = e.emission_line['la1']
+                else:
+                    # only the first line for M
+                    e_cen = e.emission_line['ma1']
+                return e_cen
+
+            e1_cen = get_line_energy(element_line1)
+            e2_cen = get_line_energy(element_line2)
+
+            # no '-' allowed in prefix name in lmfit
+            pre_name = 'pileup_' + elemental_line.replace('-', '_') + '_'
+            gauss_mod = ElementModel(prefix=pre_name)
+            gauss_mod.set_param_hint('e_offset',
+                                     value=self.compton_param['e_offset'].value,
+                                     expr='e_offset')
+            gauss_mod.set_param_hint('e_linear',
+                                     value=self.compton_param['e_linear'].value,
+                                     expr='e_linear')
+            gauss_mod.set_param_hint('e_quadratic',
+                                     value=self.compton_param['e_quadratic'].value,
+                                     expr='e_quadratic')
+            gauss_mod.set_param_hint('fwhm_offset',
+                                     value=self.compton_param['fwhm_offset'].value,
+                                     expr='fwhm_offset')
+            gauss_mod.set_param_hint('fwhm_fanoprime',
+                                     value=self.compton_param['fwhm_fanoprime'].value,
+                                     expr='fwhm_fanoprime')
+            gauss_mod.set_param_hint('epsilon', value=self.epsilon, vary=False)
```

Comment:
epsilon will always stay as a fixed value, not for fitting, so we don't need to use expr to set constraint.